src/wallet/gtest/test_wallet.cpp

```diff
@@ -71,12 +71,24 @@ class TestWallet : public CWallet {
     }
 };
 
-CWalletTx GetValidSproutReceive(const libzcash::SproutSpendingKey& sk, CAmount value, bool randomInputs, int32_t version = 2) {
-    return GetValidSproutReceive(*params, sk, value, randomInputs, version);
+CWalletTx GetValidSproutReceive(
+    const libzcash::SproutSpendingKey& sk,
+    CAmount value,
+    bool randomInputs,
+    int32_t versionGroupId = SAPLING_VERSION_GROUP_ID,
+    int32_t version = SAPLING_TX_VERSION)
```

Comment:
Because these parameters have defaults, it's possible that a call with 4 arguments that previously intended the last argument to be version, will now pass that as versionGroupId. I suggest just making them mandatory parameters, or perhaps having an overload so that either 3 or 5 arguments can be passed, but not 4.

Reply:
I encountered this several times, and was forced to fix them because 4 is not a valid `versionGroupId` (resulting in a parser error).

Reply:
How useful are the default values?   Without more context I tentatively vote for mandatory parameters.